utils/hash.go

```diff
@@ -19,14 +25,22 @@ func HashPass(password string) string {
 	return next
 }
 
-// func hashPass(pass []byte) ([]byte, error) {
-// 	pass, err := bcrypt.GenerateFromPassword(pass, 10)
-// 	if err != nil {
-// 		return nil, err
-// 	}
-// 	return pass, nil
-// }
+// GenerateSalt generates a salt for the password to be salted against
+func GenerateSalt() string {
+	b := make([]byte, 22)
+	for i := range b {
+		// #nosec
```

Comment:
Why are we silencing this warning? This is using `math/rand` rather than `crypto/rand` after all.

Reply:
Fixed

Reply:
Fixed